src/lib/db.js

```diff
@@ -128,6 +128,19 @@ function * truncateTables () {
   }
 }
 
+function * isConnected () {
+  const query = knex.client.config.client === 'strong-oracle'
+    ? 'SELECT 1 FROM DUAL' : 'SELECT 1'
+  return knex.raw(query)
+  .then(() => {
+    return true
+  })
+  .catch((e) => {
+    console.log('DB error: ', e)
```

Comment:
Oh yeah ... I'm going to remove this line anyway. I only added it because I was having trouble debugging locally.